### DISCUSSION TITLE
Please help me list popular gift!!

### DISCUSSION THREAD

**Discussion by iikyara** [NAIST-SE/AttractingContributors#91] [General]
### What **gift** do you want **to send**?
I want to know **popular gifts** when it's some anniversary, such as birthday, successful day of some project, mother's/father's day, and etc...

1.  Come to my fork: https://github.com/iikyara/AttractingContributors-Gifts
2. Add to my README file.

I look forward to your contributions

**Comment by sergiodl-naist** [comment]
I just added one more!
Please visit and try to ask a question or recommend one vtuber on my fork!
https://github.com/sergiodl-naist/AttractingContributors-VTubers

**Reply by iikyara** [reply]
I did it! Thank you very much!

[a comment by an automated account is not shown]
